ci-operator/step-registry/ipi/conf/fips/ipi-conf-fips-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+if [ ${FIPS_ENABLED} = "true" ]; then
+	cat >> "${SHARED_DIR}/install-config.yaml" << EOF
+fips: true
```

Comment:
I don't think this needs to be a separate step -- I remember Bruno suggested this can simply be done in https://steps.ci.openshift.org/reference/ipi-conf - it can just have an additional param and that's it.

Reply:
It seems that this is the practice that its already used: Check for example [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ovn/conf/ovn-conf-commands.sh)

Reply:
But on a second thought, I think it is better to just put it in `ipi-conf` as you suggested.